pkgs/by-name/fe/fedistar/package.nix

```
@@ -0,0 +1,90 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+
+  pnpm_10,
+  nodejs,
+
+  rustPlatform,
+  cargo-tauri,
+  wrapGAppsHook4,
+  pkg-config,
+  glib-networking,
+  webkitgtk_4_1,
+  openssl,
+}:
+let
+  pnpm = pnpm_10;
+  pname = "fedistar";
+  version = "1.11.0";
+  src = fetchFromGitHub {
+    owner = "h3poteto";
+    repo = "fedistar";
+    tag = "v${version}";
+    hash = "sha256-x7PljFqxC3Zht3ZEAZTA6/BClZ0g7VH2HpQLGKqQ8qo=";
+  };
+  frontend-build = stdenvNoCC.mkDerivation (finalAttrs: {
+    pname = "fedistar-frontend";
+    inherit version src;
+    pnpmDeps = pnpm.fetchDeps {
+      inherit pname version src;
+      hash = "sha256-RqTAb01W8/qcQonh44bT7zeeJEuLekfZW66mfT2vYYw=";
```

See https://github.com/NixOS/nixpkgs/pull/366043#discussion_r1922897404 (in case you didn't see the resolved thread)
```suggestion
      hash = "sha256-oYTF34LkHLHzYyIy70doFT0ynlCDw1Mku+9dVMgNBgY=";
```

---

There might be something non-deterministic about pnpm_10, since it gives me different hashes almost every time... I'll downgrade to pnpm_9 just in case this helps.

---

Thanks for reporting this! I can also confirm it's not reproducible. A temp directory name is randomly generated each time:
```console
nixpkgs2 ❯ diff -r 1 2                                                                                              
Only in 1/v10/tmp: _tmp_68_aae1029d54f37bf8961de885d7ec607d
Only in 2/v10/tmp: _tmp_68_d2d4b5669f851afbf4846ee67f32ca60
nixpkgs2 ❯ diff -r 1/v10/tmp/_tmp_68_aae1029d54f37bf8961de885d7ec607d 2/v10/tmp/_tmp_68_d2d4b5669f851afbf4846ee67f32ca60
nixpkgs2 ❯ 
```
I'll open a PR to fix it soon.